SDStateTableView/Classes/SDStateTableView.swift

```diff
@@ -229,6 +229,7 @@ public class SDStateTableView: UITableView {
         //
         if case SDStateTableViewState.withButton(_, _, _, _, _, _) = currentState {
             actionButton.isHidden = false
+            buttonConfigCallback(actionButton)
```

Comment:
Get access to the button config and execute to preserve that button state can't we simply do the following here?
```
        //
        if case SDStateTableViewState.withButton(_, _, _, _, let buttonConfig, _) = currentState {
            actionButton.isHidden = false
            buttonConfig(actionButton)
        } else {
            actionButton.isHidden = true
        }
```

Reply:
Yes indeed that should work as well. true.